docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,993 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def _get_top_k_discords(subseqs, min_dist, k=1):
</pre><p>I'd like to propose that we call this top-level function <code>def murlin</code>  (yes, with a "U" since it is a bit different from the original published work and is consistent with STUMPY function naming)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
Sounds good!! Will be revised in next commits.